addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -2992,6 +2992,8 @@ export class OdooEditor extends EventTarget {
         undoButton && undoButton.classList.toggle('disabled', !this.historyCanUndo());
         const redoButton = this.toolbar.querySelector('#redo');
         redoButton && redoButton.classList.toggle('disabled', !this.historyCanRedo());
+        const animateButton = this.toolbar.querySelector('.o_we_animate_text');
+        animateButton && animateButton.classList.toggle('d-none', this.document.contains(this.toolbar));
```

Comment:
I don't think this should be done here. What we want is for it not to show in the floating toolbar. The fact that said floating toolbar is in the document while the sidebar isn't is an unrelated detail that could very well change.

Reply:
Good Morning @Zynton Ageed it should be removed instead of hiding as, in our current code in v16 the animate button is appended to the toolbar, but was never removed from it, which should be handled. Also this issue is already fixed from saas16.4. So i have made the changes accordingly, you can review it Thanks!